controllers/common/index.js

```diff
@@ -79,44 +79,46 @@ function basicContent({ lang = null, customTemplate = null } = {}) {
                 /**
                  * Determine template to render:
                  * 1. If using a custom template defer to that
-                 * 2. If the response has child pages then render a listing page
-                 * 3. Otherwise, render an information page
+                 * 2. Otherwise, render a "CMS page" which handles child pages and content alike
                  */
                 if (customTemplate) {
                     res.render(customTemplate);
-                } else if (content.children) {
-                    // What layout mode should we use? (eg. do all of the children have an image?)
-                    const missingTrailImages = content.children.some(
-                        page => !page.trailImage
-                    );
-                    const childrenLayoutMode = missingTrailImages
-                        ? 'plain'
-                        : 'heroes';
-                    if (missingTrailImages) {
+                } else {
+                    let childrenLayoutMode = 'list';
```

Comment:
Quite a lot of logic here, perhaps try and extract this out so we can unit test it?

Reply:
Done – and ended up specifying in `routes.js` which pages get this new template, so we can drop support for the old "legacy" template fields without breaking older pages.